### PR TITLE
Add Date Published feature to blog

### LINKED ISSUE #17: Date_Published
Add date published for each article

### PR DESCRIPTION
**Pull request by Collins-Omariba**
# Description

Added  status ,published_date fields and publish function for the Stori model , plus a view to change the status to published with its corresponding url path.

Fixes #17 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -24,6 +25,11 @@ class Meta:
 )
 """stori is swahili for story. was thinking of using the slang version 'risto'/'riba' in there.. """
 class Stori(models.Model):
+    STATUS_CHOICES = (
+        ('drafted', 'drafted'),
```
Comment (JimmyTron) [inline blog/models.py:29]: Change to 'draft' instead

File: blog/models.py [hunk 2]
```diff
@@ -33,7 +39,13 @@ class Stori(models.Model):
     updated_at = models.DateTimeField(auto_now=True)
     status = models.IntegerField(choices=STATUS, default=0) #"""This here serves to indicate whether a stori has been published or not."""
     category = models.ForeignKey(Category,on_delete=models.PROTECT)
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES, default='drafted')
```
Comment (JimmyTron) [inline blog/models.py:42]: Use draft.

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Just a minor adjustment it all fine. 👍👍

**Review by hellen-22 — DISMISSED** [review]

File: blog/models.py (hunk 2 above)
Comment (Collins-Omariba) [reply blog/models.py:42]: Done

File: blog/models.py (hunk 1 above)
Comment (Collins-Omariba) [reply blog/models.py]: Done

**Comment by aibunny** [comment]
Nice implementation

**Comment by Collins-Omariba** [comment]
> Nice implementation

Thanks 👍

**Review by hellen-22 — APPROVED** [review]

**Pull request merged**